Remotion/Validation/UnitTests/RoleCollectors/AddingPropertyValidationRuleCollector1Test.cs

```diff
@@ -59,7 +62,8 @@ public void SetUp ()
 
       _propertyValidatorExtractorMock = new Mock<IPropertyValidatorExtractor>(MockBehavior.Strict);
 
-      _addingPropertyValidationRuleCollector = AddingPropertyValidationRuleCollector.Create(_userNameExpression, typeof(CustomerValidationRuleCollector1));
+      _addingPropertyValidationRuleCollectorType = AddingPropertyValidationRuleCollector.Create(_userNameExpression, typeof(CustomerValidationRuleCollector1));
+      _addingPropertyValidationRuleCollector = _addingPropertyValidationRuleCollectorType;
```

Comment:
The naming doesn't name it clear what the field is for. Now that I understand why we have it:

the interface-version is used because we just want to test the interface api origianlly and the name is much shorter. It is okay to replace the interface with the type in the field type. In the one or two places where we need the interface due to expicit implementations, I think would work best if we do a cast inline. The assumtion is that this is the special case, not the default case.

 Separately, we can decide if we want to verify that the interface is implemented correctly. In the past, we did this in a few places. I don't think there's a benefit to this here so we can just switch the type and keep the rest simple.